roles/marathon/tasks/conf.yml

```diff
@@ -38,11 +38,30 @@
   tags:
     - marathon
 
+- name: create marathon directory
+  sudo: yes
+  file:
+    dest: /etc/marathon
+    state: directory
+    mode: 0755
```

Comment:
for now, this is needed so that distributive checks pass. we may be able to tighten with the next version of distributive